docs/cheaha/tutorial/pytorch_tensorflow.md

```diff
@@ -0,0 +1,132 @@
+# Anaconda Environment Tutorial for PyTorch and TensorFlow
+
+The below tutorial would show you steps on how to create an Anaconda environment, activate, and install libraries/packages for machine and deep learning (PyTorch and Tensorflow) using an Anaconda environment on Cheaha. There are also steps on how to access and use the Shell Command Line Interface (CLI), and using Jupyter Notebook's Graphical User Interface (GUI) to work with these Anaconda environments. There are detailed steps here to guide your creation of a [Jupyter Notebook job.](../open_ondemand/ood_layout.md#interactive-apps)
+
+## Installing Anaconda Environments using Cheaha's Command Line Interface (CLI)
+
+To access Cheaha's CLI (shell), please follow the below steps.
+
+1. Login to [rc.uab.edu](https://rc.uab.edu)
+
+2. Create a job on Cheaha using the Interactive Apps dropdown option.![!Interactive Apps Dropdown Menu](images/interactive_dropdown.png)
+
+3. Select Jupyter Notebook, and fill out the options, as per your project needs, then click Launch.  For more information on compute needs, and a guide for selecting the right options click [here](../job_efficiency.md#estimating-compute-resources). ![!Jupyter Launch Button](images/jupyter_launch.png)
+
+4. Click the Connect to Jupyter button ![!Connect to Jupyter Button](images/connect_to_jupyt_button.png)
+
+    You will see the below interface. ![!Jupyter Notebook Landing Page](images/jupyter_landing_page.png)
+
+5. When the job has been created, on the My Interactive Sessions page, click the button in front of Host (usually colored blue) in the format >_c0000.
+![!host image](images/cheaha_shell_button.png)
+
+    This should open into a Shell CLI as shown below. ![!Cheaha Shell CLI](images/cheaha_shell_cli.png)
```

Comment:
Image definition needs to be on its own line with a blank line before and after.